sweat/io/strava.py

```diff
@@ -0,0 +1,215 @@
+"""Very thin wrapper around the Strava API v3
```

Comment:
Just a question: I'm not too familiar with the Strava api (and stravalib) but why choose this over [stravalib](https://github.com/hozn/stravalib)?

Reply:
I've played with stravalib very long time ago and it used to load data into objects without an option to serialize those. I've heard that now all objects have to_dict() methods but I did not check. At the end the Strava API is so simple that it takes very concise request call followed by json.loads()

I have opened an issue to integrate Strava more tightly in `sweat` #8